spec/index.bs

```diff
@@ -1850,7 +1865,28 @@ interface GPUQueue {
 GPUQueue includes GPUObjectBase;
 </script>
 
- - {{GPUQueue/copyImageBitmapToTexture()}}:
+{{GPUQueue/setBufferSubData()}} takes the `sourceData` contents starting from the offset `sourceOffset` and
+schedules a write operation of these contents to the `destination` buffer in the queue.
+The operation does nothing and produces an error if any of the following is true:
+
+ - `destination` texture wasn't created with `GPUTextureUsage.COPY_DST` flag in {{GPUTextureDescriptor/usage}}.
+ - `destinationOffset + size` exceeds the {{GPUBufferDescriptor/size}} of the `destination` buffer.
+ - `destination` buffer isn't in the `"unmapped"` [=buffer state=].
+ - `size` is not a multiple of 4.
+ - `size` is zero.
+ - `sourceOffset + size` exceeds `sourceData.length`.
```

Comment:
Also if `destinationOffset` isn't a multiple of 4, because IIRC Metal requires that.